src/pages/References.tsx

```diff
@@ -384,6 +400,11 @@ const ReferencesPage = () => {
                         description={"CVE-2022-36804 allows privilege escalation."}
                         url={"https://nvd.nist.gov/vuln/detail/CVE-2022-36804"}
                     />
+                    <Reference
+                        name={"CVE-2022-27925"}
+                        description={"Zimbra Collaboration Suite - Remote Code Execution"}
```

Comment:
This is probably a better description (taken from the user guide). 
```suggestion
                        description={"CVE-2022-27925 is a remote code execution vulnerability in the Zimbra Collaboration Suite (ZCS) that allows attackers to upload arbitrary files by abusing the mboximport functionality."}
```